docs/sources/zendesk.md

```diff
@@ -0,0 +1,60 @@
+# Event source for Zendesk
+
+This event source registers itself as a notification receiver in Zendesk in order to capture events such as ticket
+creations.
+
+## Prerequisite: API token
+
+An API token is required in order to let the TriggerMesh Zendesk event source create a corresponding [Target][zd-target]
+and [Trigger][zd-trigger] in your Zendesk account. To create a new API token from the Zendesk Admin interface, follow
+the instructions at [Generating a new API token ][zd-token].
+
+## Deploying an instance of the Source
+
+Open the Bridge creation screen and add a source of type `Zendesk`.
+
+![Adding a Zendesk source](../images/zendesk-source/create-bridge-1.png)
+
+In the Source creation form, give a name to the event source and add the following information:
+
+* **Email**: Email address associated with the Zendesk account.
+* [**Subdomain**][zd-subdom]: Name of the Zendesk subdomain, without the `zendesk.com` domain or `https://` scheme.
+* [**Token**][zd-token]: API token to communicate with the Zendesk API, as described in the previous section.
```

Comment:
is that the actual token or a reference to the secret containing the token ? If it is a secret than we need to explain it.

Reply:
It's a secret dropdown, not a raw value. I'll clarify.

Reply:
Done in 66b3e48